core/geometry/src/curve/internalContexts/EllipticalArcApproximationContext.ts

```diff
@@ -451,75 +448,100 @@ export class EllipticalArcApproximationContext {
     return this._arc;
   }
   /** The rigid transformation that rotates and translates the arc into the xy-plane at the origin. */
-  public get toPlane(): Transform {
-    return this._toPlane;
+  public get localToWorld(): Transform {
+    return this._localToWorld;
   }
   /** Whether the arc is amenable to sampling. */
   public get isValidArc(): boolean {
     return this._isValidArc;
   }
   /**
-   * Compute the maximum xy-distance between a segment or circular arc, and this elliptical arc.
-   * @return max approximation error, or `Geometry.largeCoordinateResult` if the error could not be computed.
+   * Compute the maximum xy-distance between an elliptical arc, and its segment or circular arc approximation.
+   * * Inputs should be in horizontal plane(s), as z-coordinates are ignored.
+   * @param arc elliptical arc being approximated. Sweep outside the extents of the approximation is ignored.
+   * @param approximation circular arc, or segment, assumed to start and end on the elliptical arc.
+   * @return details of the perpendicular measuring the max approximation error, or undefined if no such perpendicular.
+   * For each of `detailA` and `detailB`:
+   * * `point` is the end of the perpendicular on each curve
+   * * `fraction` is the curve parameter of the point
+   * * `a` is the distance between the points.
```

Comment:
which points? The point on the curve and on the approximation when max error happen?

Reply:
Yes. We are measuring error in the max norm, which is the farthest distance between nearby portions of the two curves, which means length of the longest segment perpendicular to both an arc in the approximation, and to the portion of the ellipse it approximates. As is typical for detail pairs, we store the segment start/end data in `detailA` and `detailB`, and each detail stores the length of the segment in `a`.